en/SUMMARY.md

```diff
@@ -7,6 +7,7 @@
 * [Python](https://github.com/mavlink/MAVSDK-Python#mavsdk-python)
 * [C++ (MAVSDK Core)](cpp/README.md)
   * [C++ QuickStart](cpp/quickstart.md)
+  * [C++ API changes](cpp/api_changes.md)
```

Comment:
Should this be "Release Notes & API changes"? A question, not a statement.

Reply:
Release notes is on GitHub releases, so I'd say no.